mappings/net/minecraft/resource/ResourceImpl.mapping

```diff
@@ -4,9 +4,14 @@ CLASS xj net/minecraft/resource/ResourceImpl
 	FIELD c packName Ljava/lang/String;
 	FIELD d id Lqs;
 	FIELD e inputStream Ljava/io/InputStream;
-	FIELD f metadataInputStream Ljava/io/InputStream;
+	FIELD f metaInputStream Ljava/io/InputStream;
```

Comment:
This is called `metadata` everywhere else? Should the other names be changed too?

Reply:
Hmm? I think we have simplified it at some resource-related places... Might be worthy for an issue for discussion.

Reply:
I'm talking about the other names right below this one.

Reply:
Hmm, I kept the full metadata name in interface methods, just simplified that for internal fields and methods and such for now.

Reply:
yes, I think the old name for this is fine tbh, the surrounding methods are `metadata`. Either change it everywhere or not all. Seeing your other comment using meta in fields and methods is fine tbh, and then keeping the full MetaData for class names?